components/support/nimbus-fml/Cargo.toml

```diff
@@ -17,6 +17,8 @@ askama = "0.10.5"
 textwrap = "0.14.2"
 heck = "0.3.3"
 unicode-segmentation = "1.8.0"
+url = "2.2.2"
+reqwest = { version = "0.11.4", features = ["blocking"], default-features = false }
```

Comment:
Remote `includes` become necessary once you support remote `imports`, since the reconciliation happens on the local machine— e.g. A is on a local machine in repo 1; it imports B, in repo 2; B includes C, also in repo 2; to the developer in repo 1, the C is remotely included in to B.

1. disabling remote loading until imports are working.
2. removing remote loading until imports are working.
3. address your concerns with remote loading before landing this.
4. file a followup issue.

Your two concerns are definitely valid! The mitigations are, I think, allowing customization and/or pinning of the version of the loaded file.

The code in `lookup_repo_path` assumes a `config` map which I thought would come from a configuration file, and specified on the command line. This would allow for repeatable builds / pinning to a specific ref, branch or tag as well as local development.

> loading a file remotely, then generating code using it is making me slightly uncomfortable

Supply chain attacks are real! I think where we get to is the same place as repeatable builds.

Going further, we could learn from package managers, and have a lock file. My gut is telling me that this would be overkill in this case.

Reply:
I would prefer 1/2 with a ticket to capture the context so we tackle it later. There is no need to address this right now, and I want us to see the win for the splitting up of large files 😄

Reply:
Filed #4991